src/pykiso/test_setup/dynamic_loader.py

```diff
@@ -284,14 +279,7 @@ def get_instance(self, name: str) -> AuxiliaryCommon:
         # of the current auxiliary
         start_method = getattr(inst, "start", None)
 
-        # TODO: remove it after all auxes are adapted to DTAuxiliaryInterface
-        #################################################################################
-        is_dt = isinstance(inst, pykiso.interfaces.dt_auxiliary.DTAuxiliaryInterface)
-        if not inst.is_instance and auto_start and is_dt:
-            inst.create_instance()
-            log.internal_debug(f"called create_instance on {name}")
-        #################################################################################
-        elif not inst.is_instance and auto_start:
+        if not inst.is_instance and auto_start:
```

Comment:
As all auxiliaries now inherit from the same interface, we could remove the method existence check above and below to just do:
```python
if not inst.is_instance and auto_start:
    inst.start()
```